gitbootcamp.rst

```diff
@@ -203,6 +203,33 @@ from the `core-workflow  <https://github.com/python/core-workflow>`_
 repository to backport the commit.
 
 
+.. _git_from_mercurial:
+
+Applying a Patch from Mercurial to Git
+--------------------------------------
+
+Scenario:
+
+- A Mercurial patch exists but there is no pull request for it.
+
+Solution:
+
+1. Download the patch locally.
+
+2. Create a new branch and switch to it.
+
+3. For Mac and Unix::
+
+   $ patch -p1 < /path/to/issueNNNN.patch
```

Comment:
the `patch` command was recommended because this command already existed before `git apply`, in the past, I used the `patch` command but you need to add `-p 1` to the command and if you try with `-p 0` or without this parameter, the patch is not applied.

with `git apply`, we have to give the path file and sure that the patch has been generated with `hg diff --git` if we have used `hg`. now we use `git` instead of `hg`. just apply the patch with `git apply`